packages/cozy-procedures/src/components/Documents.jsx

```diff
@@ -47,7 +60,8 @@ Documents.propTypes = {
   t: PropTypes.func.isRequired,
   router: PropTypes.shape({
     goBack: PropTypes.func.isRequired
-  }).isRequired
+  }).isRequired,
+  data: PropTypes.object
 }
 
-export default withRouter(translate()(Documents))
+export default withRouter(translate()(DocumentsContainer(Documents)))
```

Comment:
i'm in favor of this change, but I'm against having multiple ways to do it at the same time in the codebase. Can you please change this for the other components as well?